bindata/network/kuryr/006-controller.yaml

```diff
@@ -0,0 +1,268 @@
+              if not sgs['security_groups']:
+                  sg_id = neutron.create_security_group(body={
+                      'security_group': {
+                          'name': sg_name,
+                          'project_id': project_id,
+                  }})['security_group']['id']
+              else:
+                  sg_id = sgs['security_groups'][0]['id']
+
+              return sg_id
+
+
+          def create_sg_rule(sg_id, rule):
+              try:
+                  body = {
+                      'project_id': project_id,
+                      'security_group_id': sg_id,
+                  }
+                  body.update(rule)
+                  neutron.create_security_group_rule(body={'security_group_rule': body})
+              except n_exc.Conflict:
+                  pass
+
+
+          data = {'neutron_defaults': {}}
+          svc_net_id = create_network('kuryr-service-network')
+          svc_net = netaddr.IPNetwork(svc_net_cidr)
+          svc_subnet_id = create_subnet('kuryr-service-subnet', svc_net_id,
+                                        cidr=svc_net_cidr, gateway_ip=svc_net[-2])
+          data['neutron_defaults']['service_subnet'] = svc_subnet_id
+
+          pod_net_id = create_network('kuryr-pod-network')
+          if mode == 'Multitenant':
+              pod_sp_id = create_subnetpool('kuryr-pod-subnetpool', pod_net_cidrs)
+              pod_subnet_id = create_subnet('kuryr-pod-subnet', pod_net_id,
+                                            subnetpool_id=pod_sp_id)
+              # FIXME(dulek): This is missing setting DNS server for the pod subnet.
+              data['neutron_defaults']['pod_subnet_pool'] = pod_sp_id
+          elif mode == 'Subnet':
+              pod_subnet_id = create_subnet('kuryr-pod-subnet', pod_net_id,
+                                            cidr=pod_net_cidrs[0])
+          data['neutron_defaults']['pod_subnet'] = pod_subnet_id
+
+          if mode == 'namespace':
+              from_default_sg_id = create_sg('kuryr-allow-from-default')
+              from_namespace_sg_id = create_sg('kuryr-allow-from-namespace')
+              create_sg_rule(from_namespace_sg_id,
+                             {'ethertype': 'IPv4',
+                              'remote_group_id': from_default_sg_id,
+                              'direction': 'ingress'})
+              create_sg_rule(from_default_sg_id,
+                             {'ethertype': 'IPv4',
+                              'remote_group_id': from_namespace_sg_id,
+                              'direction': 'ingress'})
+              # FIXME(dulek): I should also open traffic from those groups to node and
+              #               master.
+
+              data['namespace_sg'] = {
+                  'sg_allow_from_namespace': from_namespace_sg_id,
+                  'sg_allow_from_default': from_default_sg_id,
+              }
+
+          config = ""
+          for section, options in data.items():
```

Comment:
does this have the extra information needed? like enabling the required kuryr handlers/drivers?

Reply:
Not yet, this will be handled in the the ConfigMap Jinja2 template. Here only go the options that the script generates.